client/src/components/Card/Card.css

```diff
@@ -0,0 +1,34 @@
+.card {
+	height: 169px;
+	width: 234px;
+	border-radius: 10px;
+	background-color: rgb(247,230,222);
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+	margin: 12px;
+
+}
+.title {
+	font-size: 18px;
+}
+.cardTitle {
```

Comment:
Can you make all your CSS class names all lower case and separated by dashes? I.e. `.card-title`, rather than `.cardTitle`?